plugins/cpu/meson.build

```diff
@@ -1,5 +1,4 @@
-if get_option('plugin_cpu').disable_auto_if(host_machine.system() != 'linux').require(hsi,
-    error_message: 'plugin_cpu needs hsi to be set').allowed()
+if host_machine.system() == 'linux' and get_option('hsi').allowed()
```

Comment:
can we just check hsi perhaps?  don't we not allow hsi outside of linux?

Reply:
Yup, you're completely correct -- fixed.